pkg/ebpf/tracee.go

```diff
@@ -478,7 +483,7 @@ func (t *Tracee) Init() error {
 			t.pidsInMntns.AddBucketItem(uint32(mountNS), uint32(pid))
 		}
 	} else {
-		logger.Debug("caps Requested", "error", err)
+		logger.Debug("Requesting capabilities", "error", err)
```

Comment:
Good catch.

Reply:
@AlonZivony But the `Requested()` can fail before calling `cb` func.

![image](https://user-images.githubusercontent.com/3372117/211591129-1a3244e3-2dd4-4eb3-a513-ddfe6f9dbdc4.png)

So, I think it must be a generic error related to Requested function. The error field should have enough information related to the error.

Reply:
Analysing it again, the Requested logic is confuse. It logs the cb error and return it only if !c.bypass doesn't err. I think this can be made clearer in the future. WDYT @rafaeldtinoco?